benefits/core/templates/core/index.html

```diff
@@ -2,15 +2,15 @@
 {% load i18n %}
 
 {% block title %}
-  {% translate "core.pages.index.title" %}
+  {% translate "Choose Provider" %}
 {% endblock title %}
 
 {% block headline %}
-  {% translate "core.pages.index.headline" %}
+  {% translate "Get reduced fare on public transportation when you tap to ride" %}
```

Comment:
Need to add the missing a here: `Get a reduced fare on public transportation when you tap to ride`

Reply:
db27d68e8ea00557eef536bf7c7cc855954a0156